garnett/trajectory.py

```diff
@@ -1230,6 +1232,18 @@ def image(self):
         self._assert_arrays_loaded()
         return self._check_nonempty_property('_image')
 
+    @property
+    def box(self):
+        """Access the frame boxes as a NumPy array.
+
+        :returns: frame boxes as an (M) element array
+        :rtype: :class:`numpy.ndarray` (dtype= :class:`~.Box`)
```

Comment:
Is it actually `dtype=Box`? I would have expected `dtype=object` since `Box` isn't a NumPy type.

Reply:
Right, fixed